src/gui/Src/Gui/SettingsDialog.h

```diff
@@ -109,6 +116,12 @@ private slots:
         calc_unsigned = 1
     };
 
+    enum EngineType

```

Comment:
There is already an enum for this in the bridgemain.h and locally I have a very similar change but I will see which one is better and manually merge :)